src/plugins/discover/public/application/apps/main/components/doc_table/doc_table_infinite.tsx

```diff
@@ -35,12 +35,15 @@ const DocTableInfiniteContent = (props: DocTableRenderProps) => {
     const scheduleCheck = debounce(() => {
       const isMobileView = document.getElementsByClassName('dscSidebar__mobile').length > 0;
       const usedScrollDiv = isMobileView ? scrollMobileElem : scrollDiv;
-
-      const scrollusedHeight = usedScrollDiv.scrollHeight;
-      const scrollTop = Math.abs(usedScrollDiv.scrollTop);
```

Comment:
QQ @Dmitriynj why did you use `Math.abs` here?

Reply:
I guess, it was added mistakenly.

Reply:
thx, just wanted to be sure that I did not miss anything! 👍